sentry_auth_github/client.py

```diff
@@ -35,7 +35,7 @@ def _request(self, path, access_token):
                 headers=headers,
             )
         except RequestException as e:
-            raise GitHubApiError(unicode(e), status=e.status_code)
+            raise GitHubApiError(unicode(e), status=getattr(e, 'status_code', None))
```

Comment:
Yeah, I think that's reasonable. It's not technically a problem though since it's always being explicitly overridden, but wouldn't hurt.